plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -407,10 +424,99 @@ public synchronized Optional<ConnectorOutputMetadata> finishCreateTable(
         requireNonNull(info, "info is null");
 
         tables.put(tableName, new TableInfo(info.columns(), info.properties(), info.comment()));
+        createViewWithPredicates(tableName, info.columns(), computedStatistics);
 
         return Optional.empty();
     }
 
+    private synchronized void createViewWithPredicates(SchemaTableName tableName, List<ColumnInfo> columns, Collection<ComputedStatistics> computedStatistics)
```

Comment:
can you call out to standard view creation logic to reuse already-exists checks?

Reply:
Yes, thanks!